src/components/calcite-action-bar/calcite-action-bar.tsx

```diff
@@ -192,6 +195,12 @@ export class CalciteActionBar {
       />
     ) : null;
 
+    if (rtl) {
```

Comment:
@caripizza can you move this logic into the `CalciteExpandToggle` functional component or inside `calcite-action` if it needs to be there?

Reply:
I thought this as well, and if you can get away with it, rather than using `shadowRoot.querySelector`, if you can, attach a `ref` to the button element instead, that way you don't have to perform a DOM query, which is slightly slower than using the ref.